sidebars.js

```diff
@@ -1,63 +1,87 @@
 module.exports = {
   docs: {
```

Comment:
Here is what this would look like:
![image](https://user-images.githubusercontent.com/34380806/87564578-cc5bc780-c67d-11ea-93c7-952a7220079c.png)